paket.dependencies

```diff
@@ -1,117 +1,112 @@
-framework: net6.0, net472
+framework: net8.0
 source https://api.nuget.org/v3/index.json
 
-// PRODUCTION STUFF
-nuget Microsoft.Extensions.Configuration 6.0.1
-nuget Microsoft.Extensions.Configuration.CommandLine 6.0.0
-nuget Microsoft.Extensions.Configuration.FileExtensions 6.0.0
-nuget Microsoft.Extensions.Configuration.EnvironmentVariables 6.0.1
-
-nuget Microsoft.Extensions.Http.Polly 6.0.3
+nuget Microsoft.Extensions.Http.Polly 8.0.3
 
 // For more healtchecks, look at https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks
-nuget AspNetCore.HealthChecks.SqlServer 6.0.2
-nuget AspNetCore.HealthChecks.NpgSql 6.0.2
+nuget AspNetCore.HealthChecks.SqlServer 8.0.0
+nuget AspNetCore.HealthChecks.NpgSql 8.0.0
 
 nuget AsyncEnumerator 4.0.2
-nuget Dapper 2.0.123
-nuget CsvHelper 27.2.1
+nuget Dapper 2.1.35
+nuget CsvHelper 31.0.2
```

Comment:
can you check if we still need this?

Reply:
is used in AddressRegistry.Importer.AddressMatch